packages/upswyng-web/src/components/Hotlines.tsx

```diff
@@ -1,9 +1,101 @@
+import { Container, colors } from "../App.styles";
+import { List, ListItem, ListItemText, Typography } from "@material-ui/core";
```

Comment:
```suggestion
import List from "@material-ui/core/List";
```
Use [path imports with MUI](https://material-ui.com/guides/minimizing-bundle-size/#option-1) when possible to reduce our dev bundle size.